platform/templates/kafka/kafka.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: streaming-service
+spec:
+  kafka:
+    replicas: 3
+    listeners:
+      plain: {}
+      tls: {}
+    config:
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+    storage:
+      type: ephemeral
```

Comment:
I guess my biggest concern would be with how a consumer would override this. Is the expected pattern supposed to be:
```
kafka:
  enabled: false
```

And then build our own `Kafka` resource? That seems reasonable. If we can't disable the baked in one though, I'm not sure we could override from the outside. That is, we'd have to name our own Kafka resource the same thing and hope that k8s actually overrides it in time.

Reply:
Otherwise, it would be nice to at least be able to configure the storage option from the outside, if we don't want to put consumers in full control of the `Kafka` resource definition.